types/linkifyjs/index.d.ts

```diff
@@ -1,8 +1,9 @@
 // Type definitions for linkifyjs 2.1
 // Project: https://github.com/SoapBox/linkifyjs#readme
 // Definitions by: Sean Zhu <https://github.com/szhu>
+//                 Ovidiu Bute <https://github.com/ovidiubute>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.8
```

Comment:
Why did you bump up the minimum TypeScript version?

Reply:
DefinitelyTyped has a build step that checks whether the dependencies of declarations make sense. Without this bump running `npm test` throws this error:

```
Error: linkifyjs depends on react but has a lower required TypeScript version.
    at checkTypeScriptVersions (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/src/check-parse-results.ts:75:23)
    at Object.checkParseResults [as default] (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/src/check-parse-results.ts:19:5)
    at <anonymous>
```

I checked other package declarations and concluded TypeScript needs to be at least at version 2.8 to fix this.